src/DataFramesMeta.jl

```diff
@@ -202,23 +367,17 @@ function replace_dotted!(e, membernames)
     Expr(:., x_new, y_new)
 end
 
+function exec(df, v, fun)
+    fun(map(c -> DataFramesMeta.getsinglecolumn(df, c), v)...)
+end
```

Comment:
Any reason to put this in a separate function? AFAICT it's used only from `with_helper`.

Also:
```suggestion
exec(df, v, fun) = fun(map(c -> DataFramesMeta.getsinglecolumn(df, c), v)...)
```

Reply:
Having it in a separate function seems be the only way to get compilation to work. As in, I couldn't just construct an anonymous function and call it in the same scope. So I looked into DataFrames and saw how it does `_transform`.

Reply:
OK. You mean because of the metaprogramming? Anyway I guess that's not a big deal.